Fails to build with GCC-7.2.0

With GCC 7.2.0:
```
  Running setup.py (path:/tmp/pip-build-RAieuy/pysingular/setup.py) egg_info for package pysingular
    Running command python setup.py egg_info
    running egg_info
    creating pip-egg-info/PySingular.egg-info
    writing pip-egg-info/PySingular.egg-info/PKG-INFO
    writing top-level names to pip-egg-info/PySingular.egg-info/top_level.txt
    writing dependency_links to pip-egg-info/PySingular.egg-info/dependency_links.txt
    writing manifest file 'pip-egg-info/PySingular.egg-info/SOURCES.txt'
    reading manifest file 'pip-egg-info/PySingular.egg-info/SOURCES.txt'
    writing manifest file 'pip-egg-info/PySingular.egg-info/SOURCES.txt'
  Source in /tmp/pip-build-RAieuy/pysingular has version 0.9.4, which satisfies requirement pysingular from https://pypi.python.org/packages/3c/c5/d0ceedf769459731ff6e6653a3debf60e7401a8e511792f8a633e5372a5c/PySingular-0.9.4.tar.gz#md5=603879b289cf5e2ee7ea5bfea38a3ebf
Installing collected packages: pysingular
  Running setup.py install for pysingular ...     Running command /home/jdemeyer/sage/local/bin/python2 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-RAieuy/pysingular/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-v5wDiL-record/install-record.txt --single-version-externally-managed --compile
    running install
    running build
    running build_ext
    building 'PySingular' extension
    creating build
    creating build/temp.linux-x86_64-2.7
    gcc -fno-strict-aliasing -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wno-unused -fPIC -DPYTHON_VERSION_OLDER_THREE_FIVE=1 -I/home/jdemeyer/sage/local/include/singular -I/home/jdemeyer/sage/local/include/python2.7 -c SingularModule.cpp -o build/temp.linux-x86_64-2.7/SingularModule.o
    SingularModule.cpp: In function 'PyObject* GetSingularCompletion(PyObject*, PyObject*)':
    SingularModule.cpp:140:47: error: ISO C++ forbids comparison between pointer and integer [-fpermissive]
         while( singular_complete_return[ pos ] != '\0' ){
                                                   ^~~~
    error: command 'gcc' failed with exit status 1
```

--- a/SingularModule.cpp
+++ b/SingularModule.cpp
@@ -137,7 +137,7 @@ static PyObject * GetSingularCompletion( PyObject* self, PyObject* args ){
     
     int pos = 0;
     PyObject * python_list = PyList_New(0);
-    while( singular_complete_return[ pos ] != '\0' ){
+    while( singular_complete_return[ pos ] ){
         PyList_Append( python_list, to_python_string( singular_complete_return[ pos ] ) );
         pos++;
     }

